modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/css/all_section.scss

```diff
@@ -0,0 +1,10 @@
+.all-section-empty-state {
```

Comment:
The `.all-section-empty-state` CSS class is too restrictive we are going to reuse the empty state in all sections so that we can change to something more general.

Reply:
You are right, I'll change it